src/components/frame/Extensions/layouts/ColonySidebar.tsx

```diff
@@ -68,11 +68,11 @@ const ColonySidebar = ({ txButtons, userHub }: Props) => {
         content: {
           title: formatText({ id: 'navigation.colonySwitcher.title' }),
           content: <ColonySwitcherContent colony={colony} />,
-          bottomActionProps: {
-            text: formatText({ id: 'button.createNewColony' }),
-            iconName: 'plus',
-            to: CREATE_COLONY_ROUTE,
-          },
```

Comment:
Can you add a comment containing @BETA to all your commented out code so that we know it will be used again at some point and isn't dead?